LabTools/CoreWidgets/CommandWidget.py

```diff
@@ -0,0 +1,861 @@
+# -*- coding: utf-8 -*-
+"""
+Created on May 28 2019
+
+@author: zackorenberg
+
+A GPIB commandline widget
+"""
+
+# -*- coding: utf-8 -*-
```

Comment:
repetition of this line

Reply:
Corrected, thank you for bringing this to my attention